Привет! МЕРДЖИТЬ ПОКА НЕ НАДО!!!

AndreyGos1986 opened the pull request:
Написал имплиментации, внес небольшие изменения в моделях и мапперах, dto.
Заливать в dev пока не надо, у меня ошибки по AdsMapper - 
 (java: Can't map property "String image" to "AdsImage image". Consider to declare/implement a mapping method: "AdsImage map(String value)".) 
- (брошу скрин в телегу), пока сделаю пулл на просмотр, возможно мелочь какую-то надо дописать, совместно подумаем.

AndreyGos1986 committed:
Привет!
Написал имплиментации, внес небольшие изменения в моделях и мапперах, dto.
Заливать в dev пока не надо, у меня ошибки по AdsMapper -  (java: Can't map property "String image" to "AdsImage image". Consider to declare/implement a mapping method: "AdsImage map(String value)".) - (брошу скрин в телегу), пока сделаю пулл на просмотр, возможно мелочь какую-то надо дописать, совместно подумаем.

noheroo commented on src/main/java/ru/skypro/homework/mapper/AdsMapper.java:
```diff
@@ -3,12 +3,20 @@
+import ru.skypro.homework.model.User;
 
-@Mapper
+@Mapper (componentModel = "spring")
```
я в pom добавлял -Amapstruct.defaultComponentModel=spring
насколько я понял это для того чтобы не писать в каждом маппере (componentModel = "spring"

noheroo commented on src/main/java/ru/skypro/homework/service/impl/AdsServiceImpl.java:
```diff
@@ -1,60 +1,198 @@
     @Override
     public Collection<AdsDto> getALLAds() {
-        return null;
+        List<Ads> allAds = adsRepository.findAll();
```
Бизин говорил что надо возвращать пустой список, а не исключение, т.к. это по сути не ошибка

return toAdsDtoList(adsRepository.findAll());

AndreyGos1986 committed:
Merge remote-tracking branch 'origin/dev' into feature-Gosudarev

AndreyGos1986 committed:
Привет!
Поправил по замечаниям. Думаю, надо слить эту ветку в основную и совместно покрутить имплементации, плюсом, у меня нестыки теперь по аннотациям One to one в ads image
